weather_forecast_retrieval/data/hrrr/file_loader.py

```diff
@@ -268,6 +259,20 @@ def convert_to_dataframes(self, var_map):
 
         return metadata, dataframe
 
+    @staticmethod
+    def format_column_names(dataframe):
+        """
+        Make new names for the columns as grid_y_x
+
+        :param dataframe:
+        :return: Array - New column names including the y and x GRIB pixel
+                         index. Example: grid_0_1 for y at 0 and x at 1
+        """
+        return [
+            'grid_{c[0]}_{c[1]}'.format(c=col)
```

Comment:
This could be the place to bring back the cell resolution naming over the cell index.

Reply:
There is really no benefit to it, it was just a naming convention internal to WFR and SMRF.